<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1116</width>
    <height>473</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>-20</x>
      <y>-20</y>
      <width>1150</width>
      <height>800</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#Insertar{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(11,131,120,219), stop:1 rgba(85,98,112,226));
	color:rgba(255,255,255,210);
	border-radius:5px;
}

QPushButton#Insertar:hover{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(150,123,111,219), stop:1 rgba(85,81,84,226));

}

QPushButton#Insertar:pressed{
	padding-left:5px;
	padding-top:5px;
	background-color:rgba(150,123,111,255);
}

QPushButton#btnListar{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0  #efa114, stop:1 #fccc7c );
	color:rgba(255,255,255,210);
	border:5px solid #00000;
}

QPushButton#btnListar:hover{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(150,123,111,219), stop:1 rgba(85,81,84,226));
	border: 2px solid #FFFFFF;

}

QPushButton#btnListar:pressed{
	padding-left:5px;
	padding-top:5px;
	background-color:rgba(150,123,111,255);
}

QPushButton#btnGuardar{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 #efa114, stop:1 #fccc7c );
	color:rgba(255,255,255,210);
	border:5px solid #00000;
}

QPushButton#btnGuardar:hover{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(150,123,111,219), stop:1 rgba(85,81,84,226));
	border: 2px solid #FFFFFF;

}

QPushButton#btnGuardar:pressed{
	padding-left:5px;
	padding-top:5px;
	background-color: #efa114;
}

QPushButton#btnModificar{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 #efa114, stop:1 #fccc7c );
	color:rgba(255,255,255,210);
	border:5px solid #00000;
}

QPushButton#btnModificar:hover{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(150,123,111,219), stop:1 rgba(85,81,84,226));
	border: 2px solid #FFFFFF;

}

QPushButton#btnModificar:pressed{
	padding-left:5px;
	padding-top:5px;
	background-color: #efa114;
}

QPushButton#btnConsultar{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 #efa114, stop:1 #fccc7c );
	color:rgba(255,255,255,210);
	border:5px solid #00000;
}

QPushButton#btnConsultar:hover{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(150,123,111,219), stop:1 rgba(85,81,84,226));
	border: 2px solid #FFFFFF;

}

QPushButton#btnConsultar:pressed{
	padding-left:5px;
	padding-top:5px;
	background-color: #efa114;
}

QPushButton#btnEliminar{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 #efa114, stop:1 #fccc7c );
	color:rgba(255,255,255,210);
	border:5px solid #00000;
}

QPushButton#btnEliminar:hover{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(150,123,111,219), stop:1 rgba(85,81,84,226));
	border: 2px solid #FFFFFF;

}

QPushButton#btnEliminar:pressed{
	padding-left:5px;
	padding-top:5px;
	background-color: #efa114;
}

QPushButton#btnRegistrar{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 #efa114, stop:1 #fccc7c );
	color:rgba(255,255,255,210);
	border:5px solid #00000;
}

QPushButton#btnRegistrar:hover{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(150,123,111,219), stop:1 rgba(85,81,84,226));
	border: 2px solid #FFFFFF;

}

QPushButton#btnRegistrar:pressed{
	padding-left:5px;
	padding-top:5px;
	background-color: #efa114;
}


</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>1100</width>
       <height>461</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:#e4ded3;
border-radius:30px</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>1100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(0,0,0,210);
</string>
     </property>
     <widget class="QWidget" name="horizontalWidget_3" native="true">
      <property name="geometry">
       <rect>
        <x>1030</x>
        <y>0</y>
        <width>70</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color:rgba(0,0,0,0);
	color:rgb(255,255,255);
	border-radius:1px;
	font-size:18px;
	font-family:dripicons-v2;
}

QPushButton:hover{
	background-color:rgb(49,48,53)
}

QPushButton#cerrar:hover{
	background-color:rgb(232,17,25)
}

QpushButton:pressed{
	padding-top:5px;
 	padding-left:5px;
}
</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="btnMinimizar">
         <property name="minimumSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>î€¤</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnCerrar">
         <property name="minimumSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>9</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>25</y>
       <width>200</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgba(255,255,255,255)</string>
     </property>
     <property name="text">
      <string>REGISTRO CLIENTES</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>75</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgba(0,0,0,180)</string>
     </property>
     <property name="text">
      <string>Nro identificacion:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>150</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgba(0,0,0,180)</string>
     </property>
     <property name="text">
      <string>Nombre:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>295</x>
       <y>150</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgba(0,0,0,180)</string>
     </property>
     <property name="text">
      <string>Celular:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtNroIdentificacion">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>100</y>
       <width>221</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(0,0,0,0);
border:1.5px solid #7c6e59;
border-radius:15px;
color:rgba(0,0,0,240);
padding-bottom:4px;
padding-left:4px</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Nro Identificacion</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtNombre">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>180</y>
       <width>221</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(0,0,0,0);
border:1.5px solid #7c6e59;
border-radius:15px;
color:rgba(0,0,0,240);
padding-bottom:4px;
padding-left:4px</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Nombre completo o Razon social</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtCelular">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>180</y>
       <width>251</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(0,0,0,0);
border:1.5px solid #7c6e59;
border-radius:15px;
color:rgba(0,0,0,240);
padding-bottom:4px;
padding-left:4px</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Celular</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnRegistrar">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>300</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background-color: rgb(195, 190, 180);
	color: rgb(0, 0, 0);
	border-radius:10px;
	border:1.5px solid #7c6e59;
}

QPushButton:hover{
background-color: rgb(218, 212, 202);
	border:1.5px solid #7c6e59;
}

QPushButton:pressed{
	padding-left:5px;
	padding-top:5px;
	background-color: rgb(195, 190, 180);
	border:1.5px solid #7c6e59;

}</string>
     </property>
     <property name="text">
      <string>Registrar</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnListar">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>350</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background-color: rgb(195, 190, 180);
	color: rgb(0, 0, 0);
	border-radius:10px;
	border:1.5px solid #7c6e59;
}

QPushButton:hover{
background-color: rgb(218, 212, 202);
	border:1.5px solid #7c6e59;
}

QPushButton:pressed{
	padding-left:5px;
	padding-top:5px;
	background-color: rgb(195, 190, 180);
	border:1.5px solid #7c6e59;

}</string>
     </property>
     <property name="text">
      <string>Listar</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtDireccion">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>250</y>
       <width>491</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(0,0,0,0);
border:1.5px solid #7c6e59;
border-radius:15px;
color:rgba(0,0,0,240);
padding-bottom:4px;
padding-left:4px</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Direccion</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>55</x>
       <y>225</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgba(0,0,0,180)</string>
     </property>
     <property name="text">
      <string>Direccion:</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="tblCliente">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>60</y>
       <width>511</width>
       <height>291</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:5px;
background-color: rgb(195, 190, 180);
border: 3px solid  #000000;
</string>
     </property>
     <column>
      <property name="text">
       <string>Identificacion</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Nombre</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Tipo de Documeto</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Direccion</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Telefono</string>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="btnModificar">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>400</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background-color: rgb(195, 190, 180);
	color: rgb(0, 0, 0);
	border-radius:10px;
	border:1.5px solid #7c6e59;
}

QPushButton:hover{
background-color: rgb(218, 212, 202);
	border:1.5px solid #7c6e59;
}

QPushButton:pressed{
	padding-left:5px;
	padding-top:5px;
	background-color: rgb(195, 190, 180);
	border:1.5px solid #7c6e59;

}</string>
     </property>
     <property name="text">
      <string>Modificar</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnConsultar">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>300</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background-color: rgb(195, 190, 180);
	color: rgb(0, 0, 0);
	border-radius:10px;
	border:1.5px solid #7c6e59;
}

QPushButton:hover{
background-color: rgb(218, 212, 202);
	border:1.5px solid #7c6e59;
}

QPushButton:pressed{
	padding-left:5px;
	padding-top:5px;
	background-color: rgb(195, 190, 180);
	border:1.5px solid #7c6e59;

}</string>
     </property>
     <property name="text">
      <string>Consultar</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnEliminar">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>350</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background-color: rgb(195, 190, 180);
	color: rgb(0, 0, 0);
	border-radius:10px;
	border:1.5px solid #7c6e59;
}

QPushButton:hover{
background-color: rgb(218, 212, 202);
	border:1.5px solid #7c6e59;
}

QPushButton:pressed{
	padding-left:5px;
	padding-top:5px;
	background-color: rgb(195, 190, 180);
	border:1.5px solid #7c6e59;

}</string>
     </property>
     <property name="text">
      <string>Eliminar </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_19">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>75</y>
       <width>161</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgba(0,0,0,180)</string>
     </property>
     <property name="text">
      <string>Tipo de identificacion</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cboTipoIdentificacion">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>100</y>
       <width>250</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(228, 222, 211);
border:1.5px solid #7c6e59;
border-radius:10px;
color: rgb(0, 0, 0);
padding-bottom:4px;
padding-left:4px
</string>
     </property>
     <property name="editable">
      <bool>false</bool>
     </property>
     <property name="currentText">
      <string>--Seleccionar--</string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <item>
      <property name="text">
       <string>--Seleccionar--</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>DNI</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>RUC</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="btnGuardar">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>300</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background-color: rgb(195, 190, 180);
	color: rgb(0, 0, 0);
	border-radius:10px;
	border:1.5px solid #7c6e59;
}

QPushButton:hover{
background-color: rgb(218, 212, 202);
	border:1.5px solid #7c6e59;
}

QPushButton:pressed{
	padding-left:5px;
	padding-top:5px;
	background-color: rgb(195, 190, 180);
	border:1.5px solid #7c6e59;

}</string>
     </property>
     <property name="text">
      <string>Guardar</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
